mobile/lib/modules/home/views/home_page.dart

```diff
@@ -212,10 +211,11 @@ class HomePage extends HookConsumerWidget {
                 top: selectionEnabledHook.value ? 0 : 60,
                 bottom: 0.0,
               ),
-              child: ref.watch(assetProvider).isEmpty
+              child: !assets.ready()
```

Comment:
For some reason, this will not load the `buildLoadingIndicator()`

Reply:
I think because the `.ready()` method is calling from a class so it is not reactive, perhap?